app/routers/todos.py

```
@@ -0,0 +1,108 @@
+            .filter(models.Todo.user_id == current_user.id)
+             .offset(skip)
+             .limit(limit)
+             .all())
+    return todos
+
+
+# Todoの作成
+@router.post("/todos", response_model=schemas.TodoOut, status_code=status.HTTP_201_CREATED)
+def create_todo(
+        todo: schemas.TodoCreate,
+        db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)